File: Problem_Plecakowy/Solution/Result.cs
public class Result
{
    public List<int> Ids = new List<int>{};
    public int totalWeight= 0;
    public int totalValue= 0 ;
    public override string ToString()
    {
        
        return $"Lista przedmiotów={string.Join(", ", Ids)} Total value: {this.totalValue}, Total Weight={this.totalWeight}";
    }
}

File: Java_1/Java_Lista_1/src/Result.java
import java.util.ArrayList;
import java.util.List;

public class Result {
    public List<Integer> Ids = new ArrayList<>();
    public int totalWeight = 0;
    public int totalValue = 0;

    @Override
    public String toString() {
        return "Result: Ids=" + Ids + ", Total Weight=" + totalWeight + ", Total Value=" + totalValue;
    }

}
